app/assets/stylesheets/kudos/components/_give-kudo.scss

```diff
@@ -3,8 +3,105 @@
   @extend .container;
   margin-bottom: $spacer;
   width: $give-kudo-width;
+  font-family: $font-gotham;
+  font-weight: 500;
+  text-align: center;
+}
+
+.give-kudo__modal {
+  text-align: left;
+  position: fixed;
+  z-index: 99 !important;
+  background: white;
+  border-radius: 4px;
+  padding: 35px 30px 35px 30px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.styled-kudo-button {
+  font-weight: 500;
+  font-size: 14pt;
+  color: $color-factual-orange;
+  background: transparent;
+  border-radius: 4px;
+  box-shadow:  $button-shadow;
+  border: none;
+  padding-right: 20px;
+  padding-left: 20px;
```

Comment:
You can use the same padding shorthand I mentioned earlier here! The vertical padding would be `0px`.